1_fetch_spatial.R

```diff
@@ -0,0 +1,187 @@
+source("1_fetch/src/Download_nhd.R")
+source('1_fetch/src/download_states_shp.R')
+source('1_fetch/src/fetch_nhdplus_data.R')
+
+p1_sp_targets_list <- list(
+  
+  # Lake locations (lat/lon) Fetch #
+  
+  ## Lake sf dataset from sharepoint - stakeholder provided
+  ## This target is the same as p2_saline_lakes_sf but includes the manually drawn polygon Carson Sink.
+  ## This target is created to 1/ have a second multipolygon dataset for lakes 2/ append Carson Sink to p2_saline_lakes_sf
+  ## This should be manually downloaded to local 1_fetch/in/ folder 
+  tar_target(
+    p1_saline_lakes_bnds_sf,
+    st_read('1_fetch/in/SalineLakeBnds.shp') %>% 
```

Comment:
This `in` folder could use some organization, or a note on the README with everything that should be in it from the sharepoint folder. In a future PR, let's package up the files going into `in` so they are in one place that can be easily downloaded to plop into the subdirectory before someone new runs the pipeline.